crates/matrix-sdk-ui/src/timeline/inner/mod.rs

```diff
@@ -1244,6 +1315,17 @@ pub(super) struct HandleManyEventsResult {
     pub items_updated: u64,
 }
 
+impl Add for HandleManyEventsResult {
```

Comment:
Please no. Explicit is better than implicit. This is not C++, let's not override operators when we don't have to.

Reply:
Where's the fun then? 😉